### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.1.1.1a - Text alternatives

### ISSUE THREAD

**Issue by kreidewe** [benjaminkott/bootstrap_package#1273]
**Page**
-
https://bitv-v12.live.typo3.cps.321.works/

**Description**
-
All non-text content that is presented to the user has a text alternative that serves the equivalent purpose.
[To WCAG](https://www.w3.org/TR/WCAG21/#non-text-content)

**Solution**
-
**Header**
Add alt-text to logo: Go to template "Introduction Package" in the TYPO3 backend. Add descriptive text alternative to constant "page.logo.alt": "TYPO3 CMS Introduction Package - Back to home"

**Footer**
Make social links in footer accessible for screen reader users. Add aria-hidden=""true" to span element containing the icon set span element containing the text to visually-hidden.

**Context, links and screenshots**
-
https://brand.wwu.edu/accessibility/guide/graphics-used-links-and-buttons-need-accessible-names

**Comment by alboth** [comment]
... there's maybe still a debete need, if it would be more Accessibel to write **Typo3 instead of TYPO3**, since it is no abbrevation @alexakressmann

**Comment by SarahEdding** [comment]
In principle, I agree with you, @alboth 
TYPO3 is read as t y p o 3, so giving typo3 instead, COULD bring the screen reader closer to natural speech. There are two reasons why I would still advise against doing this: 

1. We cannot guarantee, that all screen readers will interpret "typo3" as a word and read it phonetically correctly. Screen readers rely on lists of vocabulary, and these do not contain TYPO3.
2. Most of the introduction content is visible, so we have to spell the brand name TYPO3 correctly, there. If we adjust alt-attributes and other text alternatives to screen reader behaviour, but not the regular page content, we create a confusing situation. Screen reader users will hear "typo3" in some contexts and "t y p o 3" in others, potentially on the same subpage. If we spell the name consistently, at least it will be recognisable across the site and all its content.

**Comment by SarahEdding** [comment]
Header logo is done; social-links in footer still need aria-hidden="true" on icon-span: 

```
<a class="sociallinks-link sociallinks-link-facebook" href="https://www.facebook.com/typo3/" title="TYPO3 on Facebook" rel="noopener" target="_blank">
                            <span class="sociallinks-link-icon bootstrappackageicon bootstrappackageicon-facebook" aria-hidden="true"></span>
                            <span class="sociallinks-link-label">TYPO3 on Facebook</span>
                        </a>
```

**Comment by benjaminkott** [comment]
All icons were already adjusted, please check the header and adjust the settings on the preview system
